docs/source/analysis.ipynb

```diff
@@ -0,0 +1,510 @@
+    "\n",
+    "- **Specify the model name (the model directory name) for the model to analyze**. This notebook can be used to analyze multiple models and the model name specified here is the model that gets analyzed and visualized.\n",
+    "- **Specify the filename to the progress file**. No change is needed if the notebooks are in the project folder.\n",
+    "- **Run the following cell**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from keypoint_moseq.analysis import track_progress\n",
+    "\n",
+    "model_dirname = 'model_dir' # model directory name for the model to analyze\n",
+    "project_dir = './' # project directory\n",
+    "progress_filename = 'progress.yaml' # progress file name\n",
+    "\n",
+    "progress_paths = track_progress(model_dirname, project_dir, 'progress.yaml')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Assign Groups\n",
+    "\n",
+    "Sessions can be given group labels for analyses comparing different cohorts or experimental conditions and the labels will be stored in the `index.yaml` file.\n",
+    "\n",
+    "The following cell creates the `index.yaml` file if necessary, runs the widget to assign groups and updates group information to the `index.yaml`.\n",
+    "\n",
+    "**Instructions:**\n",
+    "- **Run the following cell**.\n",
+    "- **Click the column header** to sort the column and use the filter icon to filter if needed.\n",
+    "- **Click on the session** to select the session. **To select multiple sessions, click the sessions while holding down the [Ctrl]/[Command] key, or click the first and last entry while holding down the [Shift] key.**\n",
+    "- **Enter the group name in the `Desired Group Name` field** and click `Set Group` to update the `group` column for the selected sessions.\n",
+    "- Click the `Update Index File` button to save current group assignments.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from keypoint_moseq.wrappers import interactive_group_setting\n",
+    "progress_paths = interactive_group_setting(progress_paths)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "# Compute Syllable Satistics"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compute `moseq_df`\n",
+    "The following cell generates a `DataFrame` of kinematic values for each frame and each frame is aligned to its model label. All sessions are concatenated to form this `DataFrame`. Each row is one \"frame\" which corresponds to one sample from the original video. Each column is a different entry.\n",
```

Comment:
This needs to be updated. Refers to "extraction" which isn't part of kp-moseq and also talks about "depth"

updated here: 711f550adaf1ccca131b30ccda27ceac3afbc909